test/Spec/GT/Prelude.hs

```diff
@@ -1,8 +1,23 @@
 module Spec.GT.Prelude
     ( module GT.Prelude
+    , module Test.Hspec.Expectations.Lifted
     , module Test.Hspec
     ) where
 
 import GT.Prelude
 
-import Test.Hspec
+import Test.Hspec                     hiding
+    ( shouldBe
+    , shouldContain
+    , shouldEndWith
+    , shouldMatchList
+    , shouldNotBe
+    , shouldNotContain
+    , shouldNotReturn
+    , shouldNotSatisfy
+    , shouldReturn
+    , shouldSatisfy
+    , shouldStartWith
+    , expectationFailure
+    )
+import Test.Hspec.Expectations.Lifted
```

Comment:
Lifted expectations are nice, because they can be used directly in the `SqlPersistT` monad.